sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -299,6 +299,28 @@ private[sql] trait SQLTestUtilsBase
     }
   }
 
+  /**
+   * Drops cache `cacheName` after calling `f`.
+   */
+  protected def withCache(cacheNames: String*)(f: => Unit): Unit = {
```

Comment:
Nit: `cachedName` => `cachedTableNames`

Reply:
Hi @gengliangwang, thanks for the review. However, it should be the cache name instead of cached table name. It is because we may create a cache with a name different from corresponding table name. They are not necessary the same.

Reply:
Hi @gengliangwang , may I mark this conversation resolved?